com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -478,9 +477,7 @@ static class RenderPassBlock
 
         public ScriptableRenderer(ScriptableRendererData data)
         {
-#if URP_ENABLE_DEBUG_DISPLAY
             DebugHandler = new DebugHandler(data);
```

Comment:
Is the DebugHandler and resources tied to that, stripped from builds when doing release builds?

Reply:
This define was a temporary one meant for landing the first PR, and it's now removed. I think this is a mistake - creating DebugHandler is unnecessary in release builds so I'll remove it (it only contains a single shader in terms of resources, but still). Shader stripping happens here: https://github.com/Unity-Technologies/Graphics/blob/a597ff26b25d7a3628d021760c930bb3779f99cb/com.unity.render-pipelines.universal/Editor/ShaderPreprocessor.cs#L164

Reply:
Great great great!